modules/api/src/main/java/org/eclipse/xpanse/api/config/GetCspInfoFromRequest.java

```diff
@@ -146,9 +145,9 @@ public Csp getCspFromServicePolicyId(String id) {
      * @param taskId id of migrate task.
      * @return csp.
      */
-    public Csp getCsFromWorkflowTaskId(String taskId) {
+    public Csp getCspFromWorkflowTaskId(UUID taskId) {
         try {
```

Comment:
The taskId in workflow is not a UUID. Just keep type String

Reply:
updated